js/main.js

```diff
@@ -58,37 +105,47 @@ function entryTree(entry) {
 
   var block = document.createElement('div');
   block.className = 'row entry';
+  block.setAttribute('id', entry.entryId);
   block.appendChild(halfImg);
   block.appendChild(halfText);
   return block;
 }
-window.addEventListener('DOMContentLoaded', handleEntry);
+
 function handleEntry(event) {
-  var $ul = document.querySelector('ul');
   for (var i = 0; i < data.entries.length; i++) {
     $ul.appendChild(entryTree(data.entries[i]));
   }
 }
+window.addEventListener('DOMContentLoaded', handleEntry);
 
-if (data.entries.length === 0) {
-  document.querySelector('.no-entries').className = 'no-entries';
-
-} else {
-  document.querySelector('.no-entries').className = 'no-entries hidden';
+function handleEditing(entry) {
+  document.querySelector('.title').textContent = 'Edit Entry';
+  $photoUrl.value = entry.photoUrl;
+  $img.setAttribute('src', $photoUrl.value);
+  $entryTitle.value = entry.title;
+  $entryNotes.value = entry.notes;
+  return entry.entryId;
 }
 
 function handleClick(event) {
   if (event.target.matches('.entries-tab')) {
     switchView('entries');
   } else if (event.target.matches('.new-button')) {
     switchView('entry-form');
+  } else if (event.target.className === 'edit-icon fas fa-pen fa-2x') {
+    var curObject = data.entries.filter(j => String(j.entryId) === String(event.target.id))[0];
```

Comment:
Hmmmm, do you know how this works? If not, I suggest you try doing this on your own without the `filter` method